squad/core/models.py

```diff
@@ -494,13 +494,22 @@ class Test(models.Model):
     name = models.CharField(max_length=256, db_index=True)
     result = models.NullBooleanField()
     log = models.TextField(null=True, blank=True)
+    known_issue = models.ForeignKey('KnownIssue', null=True, blank=True)
 
     def __str__(self):
         return "%s: %s" % (self.name, self.status)
 
     @property
     def status(self):
-        return {True: 'pass', False: 'fail', None: 'skip'}[self.result]
+        if self.result:
+            return 'pass'
+        elif self.result is None:
+            return 'skip'
+        else:
+            if self.known_issue_id:
```

Comment:
yes, this is an attempt at avoiding a database query here. every foreign key field has an implicit, associated `*_id` field. this will probably have to go away (see below)